README.md

```diff
@@ -53,12 +53,18 @@ example of the `lcov` format [here](./lcov.md).
 cairo-coverage path/to/trace/1.json path/to/trace/2.json path/to/trace/3.json
 ```
 
-### Coverage statistics
+### Usage locally
 
-Various tools exist that can produce coverage statistics based on an lcov file. One of the most well-known among these
-is [genhtml](https://github.com/linux-test-project/lcov/blob/master/bin/genhtml),
-a tool that generates an html summary report using the coverage data in an lcov file, `genhtml` is part of
-the [lcov package](https://github.com/linux-test-project/lcov/tree/master).
+A summary report with aggregated data can be produced by one of many tools that accept the `lcov` format.
+In this example we will use the `genhtml` tool from the [lcov package](https://github.com/linux-test-project/lcov/tree/master) to generate an HTML report.
+
+Run the following command in the directory containing your `coverage.lcov` file:
```

Comment:
nit: as this is our only docs let's maybe reiterate that the report must be first generated (and link to the section)? not sure though

Reply:
Added 🫡